articles/openshift/oshift-how-build-app-private-repo.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to build an OpenShift application from a private Github repo | UKCloud Ltd
+description: Shows how to build an OpenShift application where the source code is in a "private" github repository secured with an SSH Key Pair.
+services: openshift
+author: Andy Watkins
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Build an OpenShift app from a private Github repo
```

Comment:
I'd spell application out in full here, but I think it's probably OK to leave repo as is